integration-tests/tests/example-page.cy.ts

```diff
@@ -0,0 +1,52 @@
+import { checkErrors } from '../support';
+
+const PLUGIN_TEMPLATE_NAME = 'console-plugin-template';
+const PLUGIN_TEMPLATE_PULL_SPEC = Cypress.env('PLUGIN_TEMPLATE_PULL_SPEC');
+const isLocalDevEnvironment = Cypress.config('baseUrl').includes('localhost');
+
+if (!Cypress.env('OPENSHIFT_CI') || Cypress.env('PLUGIN_TEMPLATE_PULL_SPEC')) {
+  describe('Console plugin template test', () => {
+    before(() => {
+      cy.login();
+
```

Comment:
> @jcaianirh I tried to load the helm chart manifest here and passed it to `cy.exec` similar to what is done in demo-plugin but that didn't work. I am not sure how to or where to pass the manifest content to, which I think it could fix the error in the next comment. Any ideas?
> 
> ```
>  cy.readFile('../charts/openshift-console-plugin/values.yaml').then(
>         (textManifest) => {
>           const yamlManifest = loadAll(textManifest);
> ```

Hmm, this might not be the issue, it seems the `helm upgrade -i` cmd is installing the content of `charts/openshift-console-plugin` which has the manifest.